tests/target/impls.rs

```diff
@@ -135,9 +131,7 @@ mod m {
 
     impl<T> PartialEq for S<T>
     where
-        T: PartialEq,
-    {
-    }
+        T: PartialEq, {}
```

Comment:
I think these changes are incorrect - we should only use the `{}` form (c.f., with a newline) if the whole impl fits on one line. Also, we should not have a trailing comma if there is a brace on the same line.